api/WorkerGlobalScope.json

```diff
@@ -0,0 +1,491 @@
+{
+  "api": {
+    "WorkerGlobalScope": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerGlobalScope",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "3.5"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": "10.6"
+          },
+          "opera_android": {
+            "version_added": "11.5"
+          },
+          "safari": {
+            "version_added": "4"
+          },
+          "safari_ios": {
+            "version_added": "5.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "caches": {
```

Comment:
Let's remove caches as we currently document it on WindowOrWorkerGlobalScope. (This whole moving around of docs because of mixin changes is a problem and leaves us confused where we should put it on BCD, too. For those purpose of migrating data, I'm trying to stick to the MDN page structures for now, but it probably needs a better structure on MDN in the first place).

Reply:
👍